apps/src/studio/studio.js

```diff
@@ -4505,6 +4483,7 @@ Studio.setMap = function (opts) {
   // Remember the requested name so that we can reuse it next time the
   // background is changed.
   Studio.wallMapRequested = opts.value;
```

Comment:
It's just used in one spot, but refactoring that away started to get pretty hairy. Ultimately it might be easier to just migrate starwars over to this style of wall and delete that code.